editor/audio_stream_preview.h

```diff
@@ -75,6 +75,15 @@ class AudioStreamPreviewGenerator : public Node {
 			thread = p_rhs.thread;
 			return *this;
 		}
+		Preview(const Preview &p_rhs) {
+			preview = p_rhs.preview;
+			base_stream = p_rhs.base_stream;
+			playback = p_rhs.playback;
+			generating.set_to(generating.is_set());
+			id = p_rhs.id;
+			thread = p_rhs.thread;
+		}
+		Preview() {}
```

Comment:
Is this unrelated? or is it just that contained types now need default and copy constructors?

Reply:
yeah the safe flag required it